src/sql/src/plan/statement/ddl.rs

```diff
@@ -595,16 +597,95 @@ pub fn plan_create_source(
             let details =
                 ProtoPostgresSourceDetails::decode(&*details).map_err(|e| sql_err!("{}", e))?;
 
+            // Register the available subsources
+            let mut available_subsources = HashMap::new();
+
+            for (i, table) in details.tables.iter().enumerate() {
+                let name = FullObjectName {
+                    database: RawDatabaseSpecifier::Ambient,
```

Comment:
When I first wrote it I thought I didn't have this info but it's in the connection. Will adapt it to also have the database so that users can fully specify their objects if they wish